package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "node test/index.js"
+    "test": "./clean.sh && node test/index.js"
```

Comment:
Can the test script clean up after itself instead? Maybe even just using fs?

Reply:
I find cleaning up before is more robust because in many cases random failures or early exits can put you in a bad state if you clean up after. Since these files are gitignored it doesn't matter if they exist when developing, we just need to be sure the space is clean before running tests.